results.tex

```diff
@@ -73,10 +73,13 @@ \subsection{Stochastic Jittering}
   \begin{subfigure}[b]{.5\columnwidth}
     \centering
     \includegraphics[width=\textwidth]{cactus_holland_jittering.png}
-    \caption{With jittering enabled.}
+    \caption{Jittering enabled.}
     \label{fig:jittering_with}
   \end{subfigure}
-  \caption{Stochastic jittering.}
+  \caption{Holland cactus dataset \emph{needCitation} exhibiting wood-grain
```

Comment:
I will need to check with the person who gave it to me, I had forgotten about that file. It didn't used to load, and I had been meaning to circle back to it. Let me email him.